data/cybox/nozomi/nozomi_alerts_22022024.json

```diff
@@ -0,0 +1,563 @@
+                    "dst_ip_ref": "1",
+                    "src_ip_ref": "3",
+                    "x_alert_status": "open",
+                    "name": "New global MAC vendor",
+                    "ioc_refs": [
+                        "8"
+                    ],
+                    "ttp_tagging_refs": [
+                        "9"
+                    ],
+                    "description": "A new Private MAC Address has been found in the network -- 52:xx:00:12:xx:02",
+                    "x_is_incident_alert": false,
+                    "x_is_cybersecurity_alert": true,
+                    "finding_type": "alert",
+                    "start": "2024-01-09T09:19:33.000Z",
+                    "x_sensor_host": "Demo Sensor standard 4959963c",
+                    "x_sensor_interface": "em1",
+                    "x_alert_type_id": "VI:GLOBAL:NEW-MAC-VENDOR"
+                },
+                "1": {
+                    "type": "ipv4-addr",
+                    "value": "5.5.5.5",
+                    "resolves_to_refs": [
+                        "4"
+                    ],
+                    "x_nozomi_info_ref": "6"
+                },
+                "2": {
+                    "type": "network-traffic",
+                    "dst_ref": "1",
+                    "src_ref": "3",
+                    "dst_port": 22,
+                    "src_port": 49739,
+                    "protocols": [
+                        "ssh",
+                        "tcp"
+                    ]
+                },
+                "3": {
+                    "type": "ipv4-addr",
+                    "value": "1.1.1.1",
+                    "resolves_to_refs": [
+                        "5"
+                    ],
+                    "x_nozomi_info_ref": "7"
+                },
+                "4": {
+                    "type": "mac-addr",
+                    "value": "52:xx:00:12:xx:02",
+                    "x_nozomi_info_ref": "6"
+                },
+                "5": {
+                    "type": "mac-addr",
+                    "value": "08:xx:27:85:xx:cd",
+                    "x_nozomi_info_ref": "7"
+                },
+                "6": {
+                    "type": "x-nozomi-info",
+                    "zone": "ProdNet-A",
+                    "roles": "producer",
+                    "label": "plc163.ACME0.corporationnet.com",
+                    "is_public": false,
+                    "is_node_learned": true,
+                    "is_reputation_bad": false
+                },
+                "7": {
+                    "type": "x-nozomi-info",
+                    "zone": "ProdNet-A",
+                    "roles": "consumer",
+                    "is_public": false,
+                    "is_node_learned": true,
+                    "is_reputation_bad": false
+                },
+                "8": {
+                    "type": "mac-addr",
+                    "value": "52:xx:00:12:xx:02"
```

Comment:
Please use a valid but clearly false MAC address, such as "01:01:01:01:01:01"

Reply:
Updated as per the suggestion.